knowledge-base/export-a-report-from-telerik-report-server-via-dot-net-client.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Export a report from Telerik Report Server via .NET Client
+description: Render a report to another format from Telerik Report Server via .NET Client.
+type: how-to
+page_title: Download a report from Telerik Report Server via .NET Client
+slug: export-a-report-from-telerik-report-server-via-dot-net-client
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Products</td>
+			<td>Progress® Telerik® Reporting, Report Server</td>
+		</tr>
+	</tbody>
+</table>
+    
+## Description
+
+[Telerik Report Server](https://www.telerik.com/report-server) has its REST API exposed, allowing you to consume resources from third-party applications as an administrator, after logging in. You can find *examples* with JavaScript client in the REST API section of the online documentation:  
+
```

Comment:
The description does not explain what we are about to do, why we are doing it and why this is not a straightforward scenario. Share more details on the actual description instead of general information about Telerik Report Server.